kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -1,26 +1,25 @@
 <template>
 
-  <core-base :topLevelPageName="topLevelPageName" @scroll="handleScroll">
+  <core-base :topLevelPageName="topLevelPageName">
     <div slot="app-bar-actions">
       <channel-switcher @switch="switchChannel"/>
-
       <ui-icon-button
         icon="search"
         type="secondary"
         color="white"
         :ariaLabel="$tr('search')"
         @click="toggleSearch"/>
     </div>
-    <breadcrumbs slot="above" class="breadcrumbs"/>
-    <component class="content" slot="content" :is="currentPage"/>
+    <div slot="content">
+      <section-nav />
```

Comment:
that's a matter of opinion :) back in XML days [the space was required](https://www.w3.org/TR/xhtml1/#guidelines).

Now it doesn't matter. I'll remove it because that seems to be more common in our code